src/goto-symex/renaming_level.h

```diff
@@ -56,7 +56,14 @@ struct symex_level0t
 /// This is to preserve locality in case of recursion
 struct symex_level1t
 {
-  symex_renaming_levelt current_names;
```

Comment:
This deletion should be in the next commit (" Make current_names private in symex_level1t"), I think